File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

a = int(input("Enter a number: "))
for i in range(1, a + 1):
    for j in range(a - i):
        print(" ", end="")
    for j in range(i * 2 -1):
        print("*", end="")
    print()

File: week-07/day-1/draw-pyramid.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

for(let i = 0; i < lineCount; i++){
    let pyramid = '';
    for(let j = 0; j < lineCount - i - 1; j++){
        pyramid += ' ';
    }
    for(let j = 0; j < i * 2 + 1; j++){
        pyramid += '*';
    }
    console.log(pyramid);
}